js/ccf-app/src/scripts/build_bundle.ts

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+import {
+  readdirSync,
+  statSync,
+  readFileSync,
+  writeFileSync,
+  existsSync,
+} from "fs";
+import { join, posix, resolve, sep } from "path";
+
+function getAllFiles(dirPath: string): string[] {
+  const toSearch = [dirPath];
+  const agg = [];
+
+  for (const filePath of toSearch) {
+    if (statSync(filePath).isDirectory()) {
+      for (const subfile of readdirSync(filePath)) {
+        toSearch.push(join(filePath, subfile));
+      }
+    } else {
+      agg.push(filePath);
+    }
+  }
+  return agg;
+}
+
+function removePrefix(s: string, prefix: string): string {
+  if (s.startsWith(prefix)) {
+    return s.slice(prefix.length).split(sep).join(posix.sep);
```

Comment:
Can this be made platform-independent by grabbing a more generic path separator?

Reply:
Now resolved.
For posterity `path.sep` chooses the relevant separator depending on the operating system.